### PR TITLE
✨ add helper:is-empty

### LINKED ISSUE #26: A new `is-empty` helper should be added
This should use the `Ember.isEmpty` function, and consider changing `truthConvert` to no longer be falsy for empty arrays. That's not how JS works, and truth helpers aren't block helpers. Consider encourage users to use `(is-empty value)` or `(not (is-empty value))` to check arrays. This change would require a semver major bump.

### PR DESCRIPTION
**Pull request by jamesarosen**
Closes #26
See also #51
Closes #57

### CODE REVIEW

**Comment by jmurphyau** [comment]
Hey

Sorry I never managed to get back on this as fast as I said I was going to on Slack the other week. Hopefully I can find some time next week to sort this out and merge it in.

Thanks for contributing!

(:

File: addon/helpers/is-empty.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { helper } from '@ember/component/helper';
+import { isEmpty } from '@ember/utils';
+
+export default helper(function([obj]) {
```
Comment (btecu) [inline addon/helpers/is-empty.js]: Could it just be `export default helper(([obj]) => isEmpty(obj));`?

Comment (jamesarosen) [reply addon/helpers/is-empty.js]: It sure could. Or even
```js
export default helper(compose(first, isEmpty))
```
if you want to go super-functional-programming. I tried to stay stylistically similar to the existing code in this repo.

**Comment by jmurphyau** [comment]
Finally found some time. going to merge this in now and (hopefully) push 2.1 over the next few days. 

Theres a few additional changes I want to make to the base files (files generated by `ember addon [name]`) to align them with the current addon template.

Thanks for this!

**Comment by jrowlingson** [comment]
@jmurphyau 👋 any time to cut that release?

**Pull request merged**